3-extensions/serialization/dubbo-samples-serialization/dubbo-samples-serialization-java/case-versions.conf

```diff
@@ -20,6 +20,6 @@
 # Supported component versions of the test case
 
 # Spring app
-dubbo.version= 3.3.*
+dubbo.version= [ < 3.3]
```

Comment:
We should also duplicate this case and refer to the implication of which in dubbo-spi-extensions.

Reply:
OK，I will move to spi-samples